tsdb/engine/tsm1/file_store.go

```diff
@@ -121,6 +124,13 @@ func (f *FileStore) Count() int {
 	return len(f.files)
 }
 
+// Files returns TSM files currently loaded.
+func (f *FileStore) Files() []TSMFile {
```

Comment:
This shouldn't be exported.  It's not safe to access the files slice outside of the `FileStore`.